### ISSUE TITLE
Host delisted from hostdb

### ISSUE THREAD

**Issue by betaboot** [NebulousLabs/Sia#1374]
I'm not sure if this is an issue but my host is not in the hostdb list anymore. I was getting new uploaded data the whole day but then the data stream stopped. Since around that time i cannot see my host in the list anymore. Some hours later i checked siapulse.com and saw that i'm listed in the 24h list but not anymore in the 6h list.

I think that happened the second time already, the first time i manually reannounced my host and then it was back. This time i will let it run to see if it comes back itself.

Anything i can do or check to get more info for you? The host.log stops around that time, but i think no unusual messages.
[host.txt](https://github.com/NebulousLabs/Sia/files/372885/host.txt)

**Comment by DavidVorick** [comment]
hmm that's pretty strange. Is your host changing IP addresses? It shouldn't be falling off the hostdb like that. Do you have >95% uptime?

Nothing obvious pops out to me. I'll keep thinking about it, and maybe add some logs to the hostdb so we can get an idea of why it's being removed.

**Comment by betaboot** [comment]
I can see from siapulse.com that my IP has not changed for at least a week. Also, i use DynDNS to cover IP changes.
Host has been up all the time for the last couple of days.
Now I use 1.0.1 but the last time it was with 1.0.0.
I had IPv6 activated before but last time I saw that my host is missing in hostdb i disabled IPv6 to make sure that's not the problem.

Can it be something with port forwarding? I just checked in my router settings that IPv6 is really disabled and then i saw that no ports are forwarded. I've seen that before, sometimes they are forwarded, sometimes not. Maybe a router/ISP problem on my side.
